test/helper.js

```diff
@@ -0,0 +1,16 @@
+var helper = require('../lib/helper');
+var expect = require('chai').expect;
+
+describe('helper', function() {
+  describe('validateLoopbackVersion()', function() {
+    it('should return valid loopback version', function() {
+      var version = helper.validateLoopbackVersion('2.1.3');
+      expect(version).to.equal('2.1.3');
+    });
+
+    it('should return default loopback version', function() {
+      var version = helper.validateLoopbackVersion('https://github.com/strongloop/loopback.git#2.x');
```

Comment:
Please add another test for `helper.validateLoopbackVersion('https://github.com/strongloop/loopback.git#2.x');`.